packages/regl-worldview/src/commands/Command.js

```diff
@@ -81,6 +82,37 @@ export default class Command<T> extends React.Component<Props<T>> {
     }
   }
 
+  getDrawPropFromHitmapId(id: number) {
+    const { hitmapProps } = this.props;
+    for (const drawProp of hitmapProps) {
+      if (drawProp.color) {
+        const drawPropId = getIdFromColor(drawProp.color.map((color) => color * 255));
+        if (drawPropId === id) {
+          return drawProp;
+        }
+      }
+    }
+    return null;
+  }
+
+  fireRelevantMouseHandler(id: number, e: any, ray: any, mouseEventName: MouseEventEnum) {
+    const mouseHandler = this.props[mouseEventName];
+    if (!mouseHandler) {
+      return;
+    }
+
+    const drawProp = this.getDrawPropFromHitmapId(id);
+    if (!drawProp) {
+      return;
+    }
+
+    mouseHandler(e, {
+      ray,
+      interactedObject: drawProp,
+      component: this,
```

Comment:
curious: why are we including the component?  I feel like there's not a lot one can do w/ a component instance in a callback.

Reply:
true! very unnecessary